### PR TITLE
Add wrw cpp translation

### PR DESCRIPTION
**Pull request by zjing14**

### CODE REVIEW

**Review by whchung — COMMENTED** [review]
LGTM. Would like @jerryyin to take another round of review too.

File: mlir/.clang-format [hunk 1]
```diff
@@ -1,2 +1,2 @@
 BasedOnStyle: LLVM
-AlwaysBreakTemplateDeclarations: Yes
\ No newline at end of file
+AlwaysBreakTemplateDeclarations: Yes
```
Comment (whchung) [inline mlir/.clang-format]: @zjing14 could you drop this?

Comment (zjing14) [reply mlir/.clang-format]: Done

**Comment by whchung** [comment]
@zjing14  sorry you might need to do a rebase because i just made a rebase to LLVM/MLIR tip.

**Review by whchung — COMMENTED** [review]
@jerryyin could you give this PR another go?

**Comment by jerryyin** [comment]
LGTM

**Pull request merged**